src/components/request/useRequestTableData.tsx

```diff
@@ -0,0 +1,190 @@
+import { useState, useEffect } from "react";
+import { ICell, IRow } from "../table/Table";
+import { DateTime } from "luxon";
+import { ethers } from "ethers";
+
+const hc: ICell[] = [
+  {
+    size: "xs",
+    value: "#",
+  },
+  {
+    size: "sm",
+    value: "Name",
+  },
+  {
+    size: "sm",
+    value: "Type",
+  },
+  {
+    size: "lg",
+    value: "Data",
+  },
+];
+
+function useRequestTableData(searchParams: URLSearchParams) {
```

Comment:
I think it might be if you can click between different requests in the UI -- but you might be right as it'll just render once when they go to the link.

Reply:
after talking about this, i think its ok to leave it as it is